handlers/network_handler.py

```diff
@@ -0,0 +1,20 @@
+import redis
+import json
+
+redis_client = redis.Redis(host='localhost', port=6379, db=0)
+
+
+def handle(event, context):
+    response = None
+    if event['pathParameters'] is not None:
+        response = redis_client.get(event['pathParameters']['hash']).decode()
```

Comment:
If this `redis_client.get(event['pathParameters']['hash'])` returns `None` it raises exception, doesn't it? What's the return of redis if you don't have this key on it?